src/components/SignUpForm.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+function SignUpFormHeader() {
+  return <h3>Luo uusi käyttäjä</h3>;
+}
+
+function SignUpForm() {
+  const [values, setValues] = useState({
+    firstName: '',
+    lastName: '',
+    userId: '',
```

Comment:
`userId` -> `username`.

Id would imo be something that is randomly generated in the database